.werft/workspace-integration-tests-startup-cron.yaml

```diff
@@ -0,0 +1,111 @@
+pod:
+  serviceAccount: werft
+  nodeSelector:
+    dev/workload: builds
+  imagePullSecrets:
+  - name: eu-gcr-io-pull-secret
+  volumes:
+  - name: gcp-sa
+    secret:
+      secretName: gcp-sa-gitpod-dev-deployer
+  - name: config
+    emptyDir: {}
+  containers:
+  - name: gcloud
+    image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:aledbf-go.4
+    workingDir: /workspace
+    imagePullPolicy: Always
```

Comment:
what is the reason for Always image pull policy?

Reply:
Well, I reference the https://github.com/gitpod-io/gitpod/blob/0d57f4abe3131cbe810071ccf6e3076d315f876e/.werft/run-integration-tests.yaml#L27.

Change the image pull policy to IfNotPresent.